extensions/amp-video/0.1/amp-video.js

```diff
@@ -548,6 +549,13 @@ export class AmpVideo extends AMP.BaseElement {
           }
           this.video_.appendChild(currSource);
         });
+
+        return;
+      }
+
+      // Cached by the remote video caching (amp-video[cache=*]).
+      if (source.hasAttribute('i-amphtml-video-cached-source')) {
```

Comment:
What is the src in this case?

Reply:
Discussed offline, lmk if you want any more context